app/leaderboard/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client"
+import { Rankings } from "./components"
+import Controls from "./components/Controls/Controls"
```

Comment:
You can update the import after updating the  `leaderboard/components/index.tsx` (see above).